README.md

```diff
@@ -517,6 +518,60 @@ The SAP BTP service operator project maintainers will respond to the best of the
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
+## Secret Formats
+
+### Default
+```bash
+#Credentials section
+uri: https://feature-flags.cfapps.stagingaws.hanavlab.ondemand.com
+username: c2Jzc19sYmo4MG9sanczZm9hcTg3YnJqcnl6NmJxZStwd25kbmxyZWdoL3piZWR0bGFieGhpNWdmZXcxaGM1dGYrdWhjdzBnPQ==
+password: YWFfeXZZdjY4WWwxYnNyZ1VqUGNEMWk3anZWNWpNPQ==
+
+#Additional data added by the operator
+instance_guid: bd9bda65-1405-4ea0-a716-5afeb13eca5b
+instance_name: sample-instance    
+plan: lite                        
+type: feature-flags  // The service offering name
+```
+
+### SecretKey
+Done by adding to the spec -> secretKey: YourSecretValue
+```bash
+#Credentials section
+<YourSecretValue>:
+  {
+    password: aa_hM7glY8ARNF8hGWb5OjRIt1PPgo=
+    uri: https://feature-flags.cfapps.stagingaws.hanavlab.ondemand.com
+    username: <user_name>
+  }
+
+#Additional data added by the operator
+instance_guid: bd9bda65-1405-4ea0-a716-5afeb13eca5b
+instance_name: sample-instance  
+plan: lite
+type: feature-flags // The service offering name
+```
+
+### SecretRootKey
+Done by adding to the spec -> secretRootKey: YourSecretValue
```

Comment:
To use SecretRootKey, add 'secretRootKey: your secret value' to the spec.  (add <> to your secret value, won't show in the comments.